CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Huh, it looks like CI is using a newer CMake than what stock Ubuntu 18.04 ships...  Neat, I guess

Reply:
Yeah, I think GHA runs the same CMake revisions across all of the hosted runners.